src/config/config.ts

```diff
@@ -167,6 +167,7 @@ export function initializeAPIClients() {
     cachePostMethods(GenomeNexusAPI, [], /POST$/);
     cachePostMethods(GenomeNexusAPIInternal, [], /POST$/);
     cachePostMethods(OncoKbAPI);
+    maskApiRequests(OncoKbAPI, getOncoKbApiUrl());
```

Comment:
should we add the warning/notice to the developer here?

Reply:
i think it should actually be where we CALL the api because that's where someone combing through the code would likely wind up.

Reply:
We call the oncokb API client functions in the packages (mutation mapper, variant view, etc.) as well. It may not be a good idea to add a warning/notice in those packages. Because ideally those standalone packages should work with the public oncokb API endpoints without crashing. We should revisit the packages (in a follow up PR) to make sure that they still work with the public endpoints to a certain extend.

On the other hand, we still want to warn others not to configure their client to consume the new portal backend endpoint. They should either use the public endpoints only or have a proper configuration with their own API tokens if they need the private data. I guess we can add a warning message where we call the api functions under the main `src` files. We just need to be clear that we don't want them to configure the client to consume the portal endpoint.